src/chttpd/src/chttpd_util.erl

```diff
@@ -110,3 +118,113 @@ get_db_info(DbName) ->
     catch
         _Tag:Error -> {error, Error}
     end.
+
+mochiweb_socket_set(Sock) ->
+    put(?MOCHIWEB_SOCKET, Sock).
+
+mochiweb_socket_clean() ->
+    erase(?MOCHIWEB_SOCKET).
+
+mochiweb_socket_get() ->
+    get(?MOCHIWEB_SOCKET).
+
+mochiweb_socket_check_msec() ->
+    MSec = config:get_integer("chttpd", "disconnect_check_msec", ?DISCONNECT_CHECK_MSEC),
+    % Add jitter to avoid a stampede in case of a larger number of concurrent connections
+    MSec + rand:uniform(MSec).
```

Comment:
might want the amount of jitter as a config item also. 30s-60s as a spread seems fine but becomes less reasonable if an admin configures a different value.

Reply:
Updated with a configurable jitter value defaulting to 15000 msec.